examples/frameworks/strands_demo/bedrock_agentcore/README.md

```diff
@@ -113,20 +182,35 @@ docker build \
 <!-- path-check-skip-end -->
 
 ### Run the Container Locally
+
+> **Note:** If you built the image with `--platform linux/arm64`, you do not need to specify platform again at runtime.
+
+Run the following command to view and set Access Key ID, Secret Access Key, and Session Token
+```bash
+aws sts get-session-token --duration 3600 --query 'Credentials.[AccessKeyId,SecretAccessKey,SessionToken]' --output text
+export AWS_ACCESS_KEY_ID="YOUR_AWS_ACCESS_KEY_ID_HERE"
+export AWS_SECRET_ACCESS_KEY="YOUR_AWS_SECRET_ACCESS_KEY_HERE"
+export AWS_SESSION_TOKEN="YOUR_AWS_SESSION_TOKEN_HERE"
+export AWS_DEFAULT_REGION="us-west-2"
+```
+
 <!-- path-check-skip-begin -->
 ```bash
 docker run \
   -p 8080:8080 \
   -p 6006:6006 \
-  -e NVIDIA_API_KEY \
+  -e NVIDIA_API_KEY=$NVIDIA_API_KEY \
```

Comment:
I'm wondering why it's necessary to explicitly pass like this rather than pick it up?